feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.component.html

```diff
@@ -0,0 +1,53 @@
+<ng-container *ngIf="orderEntry$ | async as orderEntry">
+  <ng-container *ngIf="isBundleBasedConfigurator(orderEntry)">
+    <ng-container *ngIf="(numberOfLineItems$ | async) > 0">
+      <div class="cx-number-items">
+        {{
+          'configurator.header.items'
+            | cxTranslate: { param: numberOfLineItems$ | async }
+        }}
+      </div>
+
+      <a role="link" (click)="toggleItems()" tabindex="0">
+        <ng-container *ngIf="hideItems">
```

Comment:
Could we declare the `<div class="cx-toggle-hide-items">` once and differentiate inside the div?
`<div class="cx-toggle-hide-items">
            <ng-container *ngIf="hideItems"> {{ 'configurator.header.show' | cxTranslate }} </ng-container>
            <ng-container *ngIf="!hideItems"> {{ 'configurator.header.hide' | cxTranslate }} </ng-container>
  </div>`

Reply:
Done!